### PR TITLE
Update to using docker-compose v2.

### PR DESCRIPTION
**Pull request by schancel**
This change updates the compose development setup to use the v2 compose spec.  This obviates the need for templated config files.

### CODE REVIEW

**Comment by mssola** [comment]
LGTM 👏

File: CHANGELOG.md [hunk 1]
```diff
@@ -4,6 +4,7 @@
 
 ### Fixes
 
+- Update the development environment for docker-compose v2.
```
Comment (schancel) [inline CHANGELOG.md]: Can-do

Comment (schancel) [inline CHANGELOG.md]: OIC.  I misunderstood the format. I thought 2.1.1 was the Upcoming version. 🤦‍♂️

**Comment by mssola** [comment]
Thanks !

**Pull request merged**